### PR TITLE
Adding a translation in Indonesian language to the documentation

### PR DESCRIPTION
**Pull request by fritssasia**
Hello I am frits sasia, this is the result of my contribution. I added a translation for documentation in Indonesian language
I hope this can help and make it easier for people who are not fluent in English

The changes I made :
1. I created a `doc-translation` folder
2. I created a `README-IND.md` file which contains a translation of this documentation in Indonesian language
3. I edited the README.md original documentation file by adding a dropdown menu which lists the languages available for this documentation (allowing users to read documentation in other languages) and added info about the `doc-translation` folder in the `project structure` section.

I will try to invite my friends, maybe someone is interested for contributing here
I hope you like it, thank you :)

ref.: #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +129,7 @@ Done! You can change not only the landing page but any pages like this way. (You
   * `templates` - HTML files used as ejs template. `publish` script converts a markdown file to HTML based on templates under this directory.
 * `dist` - Files compiled by `build` script. `deploy` script deploys a website to GitHub pages based on this directory. Do not change the files under this directory directly.
 * `server` - Files compiled by `build` script. `start` script opens local server based on this directory. Do not change the files under this directory directly.
+* `doc-translation` - The folder contains documentation in several languages written in markdown format.
```
Comment (parksb) [inline README.md]: What do you think about placing `README-IND.md` at the root level?

**Review by parksb — COMMENTED** [review]
Thank you :D

File: README.md (hunk 1 above)
Comment (fritssasia) [reply README.md]: > What do you think about placing `README-IND.md` at the root level?

I don't know, what do you think? :"

Comment (parksb) [reply README.md]: I think it would be better for accessibility. Can you move the `README-IND.md` file to the root?

Comment (fritssasia) [reply README.md]: > I think it would be better for accessibility. Can you move the `README-IND.md` file to the root?

great idea, of course I'll do it

Comment (fritssasia) [reply README.md]: @parksb I just moved the README-IND.md file to root, is this correct? What do you think? :"

I also changed the link in the README.md file (the link that will display the README-IND.md file)

Comment (fritssasia) [reply README.md]: By the way, sorry if there is a mistake
I'm not very experienced in contributing to open source projects yet😅

Comment (parksb) [inline README.md]:
```suggestion
```

Please remove this line from README.md and README-IND.md

File: README.md [hunk 2]
```diff
@@ -10,6 +10,15 @@
   </h1>
 </div>
 
+<details>
+<summary>Click here to read this documentation in other languages</summary>
+List of available languages :
+<br>
+
+  * English (default)
+  * [Indonesian](https://github.com/parksb/handmade-blog/blob/master/README-IND.md)
```
Comment (parksb) [inline README.md]:
```suggestion
  * [Indonesian](README-IND.md)
```

You can attach a path to file within the same project.

File: README.md (hunk 1 above)
Comment (parksb) [reply README.md:132]: That's good 🙂 I left two comments suggesting little changes.

Comment (parksb) [reply README.md]: That's good 🙂 I left two comments suggesting little changes.

Comment (fritssasia) [reply README.md]: @parksb Sorry for being sleepy I forgot to remove the doc-translation line, thanks for reminding me

I've finished removing the `doc-translation` line and changing the link which will display the `README-IND.md` file

**Comment by parksb** [comment]
LGTM. Thank you!

**Comment by fritssasia** [comment]
By the way, does the `README-IND.md` file need to add a dropdown menu for the available language options? as in the `README.md` file

**Comment by fritssasia** [comment]
> LGTM. Thank you!

You're welcome, thank you also for accepting me to contribute to this repository

**Comment by parksb** [comment]
> By the way, does the `README-IND.md` file need to add a dropdown menu for the available language options? as in the `README.md` file

I don't think you have to, but if you have time, please send a new PR 😉

**Pull request merged**